src/MasterSite.php

```diff
@@ -493,8 +493,19 @@ public function p4_master_theme_setup() {
 	 */
 	public function add_to_context( $context ) {
 		global $wp;
+
 		$context['cookies']      = [
-			'text' => planet4_get_option( 'cookies_field' ),
+			'text'                      => planet4_get_option( 'cookies_field' ),
+			'enable_analytical_cookies' => planet4_get_option( 'enable_analytical_cookies' ),
+			'enable_reject_all_cookies' => planet4_get_option( 'enable_reject_all_cookies' ),
+			'settings_copy'             => [
+				'necessary_cookies_name'         => planet4_get_option( 'necessary_cookies_name' ) ?? '',
```

Comment:
the function has a `$default` parameter `planet4_get_option( $key = '', $default = null )`, so you can use it like this `planet4_get_option( 'necessary_cookies_name', '' )` instead of `??`